chore: update change logs (#2027)
emqtt/emq-docs-cn: en_US/changes/changes-ee-4.4.md

  Before the fix, `mongo_date()` can be used normally, but an error will occur when testing on the SQL test page of the Dashboard.

> - Fixed the issue that the rule engine failed to send messages via RabbitMQ actions after hot-upgrading to 4.4.19.

Reviewer: ```suggestion
- Fixed the issue where the rule engine failed to send messages through RabbitMQ actions after a hot upgrade to version 4.4.19.
```

Reviewer 2: suggestion is already committed, thanks.
